.github/workflows/wheels.yml

```diff
@@ -1,47 +1,9 @@
 ---
 name: Wheel Builds
 on:
-  push:
-    tags:
-      - '*'
+  pull_request:
+    branches: [ main, 'stable/*' ]
 jobs:
-  rustworkx-core:
-    name: Publish rustworkx-core
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions-rs/toolchain@v1
-        with:
-          toolchain: stable
-          override: true
-      - name: Run cargo publish
-        run: |
-          cd rustworkx-core
-          cargo publish
-        env:
-          CARGO_REGISTRY_TOKEN: ${{ secrets.CARGO_REGISTRY_TOKEN }}
```

Comment:
If I understood well, `rustworkx-core` will not be published by the action? Is this a mistake?

Reply:
This pr is still configured for testing, once I've got everything in a good place I will revert https://github.com/Qiskit/rustworkx/pull/753/commits/31ba30ecd16a09115ae0f47f90e038e9b7e42206 and remove the on hold label